client/src/backoffice/Main.tsx

```diff
@@ -96,20 +187,111 @@ export const BackOffice: React.FC = () => (
     </header>
     <main className="app-content">
       <div className="container">
-        <Route path={ROUTES.issueForEvent} component={IssueForEventPage} />
-        <Route path={ROUTES.issueForUser} component={IssueForUserPage} />
-        <Route path={ROUTES.events} component={EventsPage} />
-        <Route path={ROUTES.minters} component={MintersPage} />
-        <Route path={ROUTES.burn} component={BurnPage} />
-        <Route path={ROUTES.addressManagement} component={AddressManagementPage} />
-        <Route path={ROUTES.transactions} component={TransactionsPage} />
-        <Route path={ROUTES.inbox} component={InboxPage} />
-        <Route path={ROUTES.inboxList} component={InboxListPage} />
-        <Route
-          exact
-          path={ROUTES.admin}
-          render={() => <div>Choose an option from the right side menu</div>}
-        />
+        <Switch>
+          <Route exact path={ROUTES.admin} component={Landing} />
+
+          <Route
+            exact
+            path={ROUTES.issueForEvent.path}
+            render={() => (
+              <CheckRole roles={ROUTES.issueForEvent.roles}>
+                <IssueForEventPage />
+              </CheckRole>
```

Comment:
We shouldn't use anymore `CheckRole` component and instead use `WithRoleHOC`.
I.e: 
```jsx
const IssueForEventPageWithRole = WithRoleHOC(IssueForEventPage)
<IssueForEventPageWithRole roles={ROUTES.issueForEvent.roles} />
```